### PR TITLE
Update project-settings.md

### PR DESCRIPTION
**Pull request by AGulev**

### CODE REVIEW

File: docs/en/manuals/project-settings.md [hunk 1]
```diff
@@ -321,6 +321,12 @@ If specified, use this info.plist file when bundling your app.
 #### Entitlements
 If specified, can override wildcard entitlements defined in the supplied provisioning profile (.entitlements, .xcent, .plist).
 
+#### Default Language
+This language uses if application doesn't have user's preferred language in `Localizations` list (see [CFBundleDevelopmentRegion](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-130430)). Use the two-letter ISO 639-1 standard if preferred language available there or the three-letter ISO 639-2.
```
Comment (britzl) [inline docs/en/manuals/project-settings.md:325]: "This language uses if application" -> "The language used if the application"

"if preferred language available" -> "If preferred language is available"

File: docs/en/manuals/project-settings.md [hunk 2]
```diff
@@ -370,6 +376,12 @@ If set, use the specified info.plist file when bundling.
 #### Bundle Identifier
 The bundle identifier lets OS X recognize updates to your app. Your bundle ID must be registered with Apple and be unique to your app. You cannot use the same identifier for both iOS and OS X apps.
 
+#### Default Language
+This language uses if application doesn't have user's preferred language in `Localizations` list (see [CFBundleDevelopmentRegion](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-130430)). Use the two-letter ISO 639-1 standard if preferred language available there or the three-letter ISO 639-2.
```
Comment (britzl) [inline docs/en/manuals/project-settings.md:380]: Same as above

Comment (AGulev) [reply docs/en/manuals/project-settings.md]: Thx, Done!

File: docs/en/manuals/project-settings.md (hunk 1 above)
Comment (AGulev) [reply docs/en/manuals/project-settings.md]: Thx, Done!

**Pull request merged**